backend/GymBeam/Controllers/UserController.cs

```diff
@@ -43,6 +47,35 @@ public ActionResult<UserResponse> Get(int id)
             };
         }
 
+        [HttpGet("LoggedIn")]
+#if !DEBUG
+        [Authorize(Roles = Roles.User)]
+#endif
+        public ActionResult<UserResponse> GetLoggedIn()
+        {
+            int userId;
+            try
+            {
+                string cookiesUserId = Request.Cookies["X-User-Id"];
```

Comment:
Could you add this x user ID to constant and replace it everywhere? The same with other cookies names